wordclock_interfaces/templates/scripts/scripts.js

```diff
@@ -0,0 +1,154 @@
+var ColorPicker = window.VueColorPicker;
+new Vue(
+{
+	el: '#app',
+	components: {
+                ColorPicker: ColorPicker
+	},
+	data: {
+		brightness: 50,
+                color: {
+                    hue: 50,
+                    saturation: 100,
+                    luminosity: 50,
+                    alpha: 1
+                },
+		apiData: undefined,
+		selectedPlugin: {
+		    name: 'time_default'
+		},
+		currentPlugin: {
+		    name: 'time_default'
+		},
+		switchWords: true,
+		switchMinutes: true,
+		switchBackground: false
+	},
+	methods: {
+		loadApi: function () {
+			this.$http.get('/api/plugin').
+			  then(this.successCallback2, this.errorCallback);
+			this.$http.get('/api/plugins').
+			  then(this.successCallback, this.errorCallback);
+		},
+		successCallback: function(response) {
+			this.apiData = response.data;
+			console.log('successCallback this.apiData:' , this.apiData);
+		},
+		successCallback2: function(response) {
```

Comment:
Please rename to make the meaning a bit clearer.....

Reply:
I will improve.